cloudinit/url_helper.py

```diff
@@ -81,14 +81,17 @@ def combine_single(url, add_on):
     return url
 
 
-def read_file_or_url(url, timeout=5, retries=10,
-                     headers=None, data=None, sec_between=1, ssl_details=None,
-                     headers_cb=None, exception_cb=None):
+def read_file_or_url(url, *args, **kwargs):
```

Comment:
As I (just now) mentioned [here](https://github.com/canonical/cloud-init/pull/66#discussion_r351319645), I think we should drop `*args` from the function signature, which will make all the parameters other than `url` keyword-only.